config/defaults.conf

```diff
@@ -234,18 +234,17 @@ bert_fine_tune = 0  // If false, BERT transformer weights will
                     // lowering the batch size if set to true.
 
 // Sentence Encoder //
-sent_enc = rnn  // Type of sentence encoder: 'bow', 'rnn' (for LSTM), or 'transformer'
-                // set to 'null' to do nothing but forward the skip connection.
+sent_enc = rnn  // Type of sentence encoder: 'bow', 'rnn' (for LSTM),, or 'transformer'
+                // set to 'null' to feed the output of the pretrained encoder into the task specific module.
                 // Note: Transformer has no known issues, but hasn't performed well, so there may be bugs.
-                // Note: 'bow' just skips the encoder step and passes the word representations to the task model, s
+                // Note: 'bow' just skips the encoder step and passes the word representations to the task model,
                 //   so it is possible to combine attention or max pooling with the 'bow' encoder.
```

Comment:
Make it clear here that a typical BERT/GPT use case should involve 'none', not 'transformer'.

Reply:
Bump.